ospo-book/chapters/03-chapter.md

```diff
@@ -1 +1,146 @@
+
+* Open Source Advocate: OSPOs can promote the use and/or contribution of open source and best practices across different organizational units. This can help organizations realize the benefits of open source as well as engaging people to contribute to open source projects or start new ones.
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+
+## Creating and Implementing an Open Source Strategy
+
+The way the people behind an OSPO achieve this is by creating and maintaining a framework covering the following aspects: strategy, governance, compliance, and community engagement. 
+The OSPO's strategy focuses on aligning the organization's open source goals with its overall organization objectives.
+
+A strategy creates a high-level consensus on concrete topics and their impact on your organization and the people within it. A good practice is to [document this strategy in an open source strategy document](https://todogroup.org/guides/strategy/).
+
+It is recommended that this document includes a general Q&A section. Additionally, you may consider creating specific how-to guides for specialized areas that interact with the OSPO and open source projects (e.g., marketing, legal, engineering) to address specific challenges. These guides should be tailored to the domain knowledge of the team members involved.
+
+### Making Open Source Relevant in the organization
+
+Effectively executed OSPO work takes into account the elements of an organization's architecture, as understanding the organization's goals is fundamental for making 
+informed open source-forward decisions:
+
+![organization-architecture](https://github.com/todogroup/ospology/assets/43671777/e722d9b0-04f7-4f0a-b645-fd369a2b56f9)
+
+Since every organization is unique in its values, business drivers, and culture, it is challenging to provide specific content. However, addressing the following questions can help structure the document effectively:
+
+* Which types of open source projects directly align with your organization's goals??
+* Which specific practices can best foster a sustainable open source ecosystem?
+* Which organization's processes have areas for improvement? 
+* How can open source support those improvements? 
+* How can you make workers champions for open source?
```

Comment:
```suggestion
* How can you make employees champions for open source?
```

Reply:
workers is a more inclusive word that can be used in different organizations, not only private companies. For instance, this is also used in the OSPO Definition: https://ospoglossary.todogroup.org/ospo-definition/